dist/ci/travis_before_script.sh

```diff
@@ -31,8 +31,3 @@ popd
 
 # travis rvm can not deal with our extended executable names
 sed -i 1,1s,\.ruby2\.4,, src/api/{script,bin}/* docs/api/restility/bin/*
-
-echo "Build apidocs"
-pushd docs/api
```

Comment:
I'm not too familiar with how the api docs are built so I may be wrong, but don't we want to include this in the script? So travis checks that the api docs build ok? Otherwise if they are broken we might not find out until somebody complains about them..